### PR TITLE
Remove OCP-44578 and add storage tag for upgrade case

### PR DESCRIPTION
**Pull request by duanwei33**
1. Add @inactive tag to disable OCP-44578 due to fix https://issues.redhat.com/browse/OCPQE-15462
2. Add @storage tag for upgrade case

### CODE REVIEW

File: features/upgrade/storage/upgrade.feature [hunk 1]
```diff
@@ -504,6 +511,7 @@ Feature: Storage upgrade tests
   @upgrade
   @network-ovnkubernetes @network-openshiftsdn
   @hypershift-hosted
+  @inactive
```
Comment (liangxia) [inline features/upgrade/storage/upgrade.feature:514]: Instead of inactive, perhaps we should remove it as the test case have been removed in Polarion.

Comment (duanwei33) [reply features/upgrade/storage/upgrade.feature]: It makes sense, updated.

File: features/upgrade/storage/upgrade.feature [hunk 2]
```diff
@@ -404,6 +410,7 @@ Feature: Storage upgrade tests
...
   Scenario: OCP-44578:Storage AWS-EBS-CSI Snapshot operator should work well before and after upgrade of a cluster - prepare
```
Comment (liangxia) [inline features/upgrade/storage/upgrade.feature:414]: The upgrade prepare should remove in pair with the upgrade check.

Comment (duanwei33) [reply features/upgrade/storage/upgrade.feature]: Aha, just realized it when send the last comment, sorry for my overlooking, and thanks for the careful review!

**Pull request merged**